packages/react/src/components/Dialog/DialogFooter.styles.ts

```diff
@@ -42,9 +42,10 @@ export const getStyles = (props: IDialogFooterStyleProps): IDialogFooterStyles =
     actionsRight: [
       classNames.actionsRight,
       {
-        textAlign: 'right',
-        marginRight: '-4px',
+        display: 'flex',
         fontSize: '0',
+        justifyContent: 'flex-end',
+        marginRight: '-4px',
```

Comment:
Would it make sense to specify `align-items` in case one action has a different height from the other one? Right now it's `normal`/`stretch`, so the inner `.ms-Dialog-action` spans will all have the same height and their content should be top-aligned, which also seems fine. I think it'd only matter if we wanted them to be center-aligned.

Reply:
We can definitely make them `alignItems: 'center'` for completeness :smile